qemu/tests/cfg/nvdimm.cfg

```diff
@@ -58,6 +58,8 @@
             variants:
                 - @pmem_default:
                 - pmem_on:
+                    only Host_RHEL.m7 Host_RHEL.m8 Host_RHEL.m9
+                    only RHEL.7 RHEL.8 RHEL.9
```

Comment:
hi @mcasquer , may I ask if RHEL.10 guest os cannot be supported if pmem=on set in qemu?  Will pmem take effect on both host os and guest os ? I am asking because in our test matrix, RHEL10 is also a supported guest OS for RHEL9.5 host.

Reply:
@zhencliu 
I missed that :grimacing: the guest is supported

Reply:
Code updated !